src/types/homeassistant/data/area_registry.ts

```diff
@@ -9,6 +9,7 @@
 export interface AreaRegistryEntry {
   area_id: string;
   name: string;
+  icon?: string;
```

Comment:
AFAIA the `icon` property is not included in Home assistant's `AreaRegistryEntry` interface.

Since it's actually a property, it should be included.
Can you find out if it's declared somewhere in Home Assistant's type definitions and adjust our type definition accordingly?

or else,
Doesn't this property always exist and set to `null` for no icon?
If so, the property should be declared as `icon: string | null`.

Reply:
Thanks for the feedback. Per https://github.com/home-assistant/frontend/blob/dev/src/data/area_registry.ts#L14, `icon` is declared as `string | null`. I'll add a commit which adjust my original change.